pjsip/include/pjsua-lib/pjsua.h

```diff
@@ -1080,6 +1080,50 @@ typedef struct pjsua_call_setting
 } pjsua_call_setting;
 
 
+/**
+ * This will contain the information of the callback \a on_rejected_incoming_call.
+ */
+typedef struct pjsua_on_rejected_incoming_call_param {
+    /** 
+     * Local URI 
+     */
+    pj_str_t        local_info;
+
+    /** 
+     * Remote URI 
+     */
+    pj_str_t        remote_info;
+
+    /** 
+     * Rejection code 
+     */
+    int             code;
+
+    /** 
+     * Original rejection message 
+     */
+    pjsip_tx_data  *tdata;
```

Comment:
Perhaps it'd be better to provide `rdata` too if possible, in case app wants to examine the original request.